src/clj/rems/auth/oidc.clj

```diff
@@ -45,7 +45,7 @@
         ;; locale – could be used to set preferred lang on first login
         ;; email – non-unique (!) email
         id-data (jwt/validate id-token issuer audience now)
-        identity-base {:eppn (:sub id-data)
+        identity-base {:eppn (get id-data (keyword (getx env :oidc-userid-attribute)))
```

Comment:
Couldn't the attribute be a keyword instead of a string in the config?

Reply:
Yeah, it could, but I picked a string so that it would map more directly to the JSON oidc sends us

Reply:
One further advantage is that a string works for a key like `":boing foo"`  which can be impossible to express as a keyword literal